src/Calypso-Browser/ClyBrowserMorph.class.st

```diff
@@ -123,7 +123,9 @@ Class {
 		'navigationStarted',
 		'plugins',
 		'navigationEnvironment',
-		'systemScope'
+		'systemScope',
+		'toolbars',
```

Comment:
It looks like you have unused i.v.

Reply:
Oh you are right, I commited that by accident!